python/cuml/ensemble/randomforestregressor.pyx

```diff
@@ -372,6 +373,37 @@ class RandomForestRegressor(Base):
         model_protobuf_bytes = save_model(<ModelHandle> model_ptr)
         return model_protobuf_bytes
 
+    def _tl_model_handles(self, model_bytes):
```

Comment:
A lot of code is duplicated between the classifier and regressor (both here and in `python/cuml/dask/ensemble`).